schema.json

```diff
@@ -599,6 +542,7 @@
           "type": "string"
         }
       },
+      "required": ["A01"],
```

Comment:
@stiebitzhofer  For now, no properties are required in `CommercialTransaction`. 
I supposed A01 should always be present, maybe you have other properties in mind (except for A06, and A06.1, A06.2 that should be a dynamically required ) ?

Reply:
Good point:A02, A03, A04, A05 and A07 should be required.